src/main/java/duke/command/DeleteFromList.java

```diff
@@ -0,0 +1,59 @@
+package duke.command;
+
+import duke.data.TaskList;
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidInputException;
+import duke.filemanager.Storage;
+import duke.task.Task;
+import duke.ui.Ui;
+
+import java.util.NoSuchElementException;
+
+/**
+ * Handles the delete command to remove task from list
+ */
+public class DeleteFromList extends Command {
```

Comment:
May want to rename this class for consistency with other methods such as MarkCommand or ExitCommand